### PR TITLE
feat: Remove PostHog event calls

### PR DESCRIPTION
**Pull request by RicardoE105**
TO-DO:
- Delete posthog-js FE hooks

### CODE REVIEW

File: packages/editor-ui/src/plugins/telemetry/index.ts [hunk 1]
```diff
@@ -188,6 +189,7 @@ export class Telemetry {
 					break;
 				case 'nodeView.addNodeButton':
 					this.track('User added node to workflow canvas', properties);
+					usePostHog().capture('User added node to workflow canvas', properties);
```
Comment (mutdmour) [inline packages/editor-ui/src/plugins/telemetry/index.ts:192]: Might be cleaner if we add a prop like usePosthog on the BE

**Review by mutdmour — COMMENTED** [review]
Would be cleaner to update track function instead to support Posthog.. what do you think?

File: packages/editor-ui/src/stores/posthog.store.ts [hunk 2]
```diff
@@ -161,10 +161,17 @@ export const usePostHog = defineStore('posthog', () => {
 		trackedDemoExp.value[name] = variant;
 	};
 
+	const capture = (event: string, properties: IDataObject) => {
```
Comment (mutdmour) [inline packages/editor-ui/src/stores/posthog.store.ts:164]: if we are going to remove the PH hooks file, we also need to make sure we add all the functionality there...
identify https://github.com/n8n-io/n8n-hosted-backend/pull/565/files#diff-3e7bba39ebab6689ae6e79d5920af1871e8fc90da4e0698af49355fc521aaf5dL18
setMetadata https://github.com/n8n-io/n8n-hosted-backend/pull/565/files#diff-3e7bba39ebab6689ae6e79d5920af1871e8fc90da4e0698af49355fc521aaf5dL59
and make sure credentials are still not captured by variables

Comment (RicardoE105) [reply packages/editor-ui/src/stores/posthog.store.ts:164]: Let's sync about this in our 1:1

File: packages/editor-ui/src/stores/posthog.store.ts [hunk 3]
```diff
@@ -161,10 +161,29 @@ export const usePostHog = defineStore('posthog', () => {
...
+		if (typeof window.posthog?.register !== 'function') return;
+
+		if (target === 'user') {
+			window.posthog?.people?.set(metadata);
+		} else if (target === 'events') {
+			window.posthog?.register(metadata);
+		}
+	};
+
 	return {
```
Comment (RicardoE105) [inline packages/editor-ui/src/stores/posthog.store.ts:181]: We said to also bring the identity from the posthog-hooks but we are already identifying in this file! 

https://github.com/n8n-io/n8n/blob/4e3b8bdb1469366e1d36d20ef7741b9be44ccf55/packages/editor-ui/src/stores/posthog.store.ts#L71

Comment (mutdmour) [reply packages/editor-ui/src/stores/posthog.store.ts:181]: perfect..

**Review by mutdmour — APPROVED** [review]

**Review by OlegIvaniv — APPROVED** [review]
Re-approving

**Comment by janober** [comment]
Got released with [`n8n@1.4.1`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.4.1)

**Pull request merged**